src/renderer/components/+network-ingresses/ingress-details.tsx

```diff
@@ -100,6 +103,10 @@ export class IngressDetails extends React.Component<Props> {
       <Trans>Network</Trans>,
       <Trans>Duration</Trans>,
     ];
+
+    const serviceName = spec?.defaultBackend?.service.name ?? spec?.backend?.serviceName;
```

Comment:
Should `Ingress` class have helper methods for `serviceName` and `servicePort`?

Reply:
Good idea, added